Runtime/Scripts/RGBotLocalRuntime/RG.cs

```diff
@@ -75,6 +94,21 @@ public RGStateEntity FindNearestEntity(string objectType = null, Vector3? positi
 
             return result.Count>0 ? result[0] : null;
         }
+        
+        /**
+         * <summary>Used to find a button Entity with the specific type name.</summary>
+         * <param name="buttonName">{string | null} Search for button entities with a specific type name.</param>
+         * <returns>{RGStateEntity} The Entity for a button matching the search criteria, or null if none match.</returns>
+         */
+        public RGStateEntity GetInteractableButton(string buttonName)
+        {
+            RGStateEntity button = FindEntities(buttonName).FirstOrDefault();
+            if (button != null && EntityHasAttribute(button, "interactable", true))
```

Comment:
I think this one should be in the botcode, not `RG` since this is specific to the BossRoom button state

Reply:
Retracting this comment